apps/deploy-web/src/components/alerts/NotificationChannelSelectForm/NotificationChannelSelect.spec.tsx

```diff
@@ -5,36 +5,83 @@ import type { FCWithChildren } from "@src/types/component";
 import { NotificationChannelSelectView } from "./NotificationChannelSelect";
 
 import { render, screen } from "@testing-library/react";
+import { buildNotificationChannel } from "@tests/seeders/notificationChannel";
 
-describe("NotificationChannelSelectView (integration with form)", () => {
-  const Wrapper: FCWithChildren = ({ children }) => {
-    const methods = useForm({ defaultValues: { notificationChannelId: "" } });
-    return <FormProvider {...methods}>{children}</FormProvider>;
-  };
+describe("NotificationChannelSelectView", () => {
+  it("renders select with placeholder when no data", () => {
+    setup({ data: [] });
 
-  const notificationChannels = [
-    { id: "1", name: "Email: alice@example.com" },
-    { id: "2", name: "Phone: +123456789" }
-  ] as NotificationChannelsOutput;
+    expect(screen.queryByText("Notification Channel")).toBeInTheDocument();
+    expect(screen.queryByText("Select notification channel")).toBeInTheDocument();
+  });
 
-  it("renders select items from data", () => {
-    render(
-      <Wrapper>
-        <NotificationChannelSelectView data={[]} isFetched={true} />
-      </Wrapper>
-    );
+  it("renders select trigger when data is provided", () => {
+    setup();
 
-    expect(screen.getByText("Notification Channel")).toBeInTheDocument();
-    expect(screen.getByText("Select notification channel")).toBeInTheDocument();
+    expect(screen.queryByText("Notification Channel")).toBeInTheDocument();
+    expect(screen.getByTestId("notification-channel-select-trigger")).toBeInTheDocument();
```

Comment:
suggestion(non-blocking): you don't need test-id here, you can rely on label:

```ts
    expect(screen.getByLabelText(/Notification Channel/)).toBeInTheDocument();
```

The first 2 methods to use for finding form fields in testing-library are:
* `getByLabelText`
* `getByPlaceholderText`

And only if for some reason this is not possible, we should add testid